### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Mathieu Rabot and Florian Duruz

### DISCUSSION THREAD

**Discussion by Nomeos** [heig-vd-dai-course/heig-vd-dai-course#334] [Show and tell]
# Repository
https://github.com/Nomeos/DAI-2024-2025-Practical-work-1-Mathieu-Rabot-and-Florian-Duruz

# Hash last commit 
de6492a3782bb9c993d8529cb84d1a0e85b2822f

**Comment by Nomeos** [comment]
## Proposition : 

### Image Modifier : On prend une ou plusieurs images en entrées et l'utilisateur a un accès a des modifications possibles (noir et blanc, fusion d'image, dilatation morphologique).

Obligatoire :
- Mettre en négatif
- Noir et blanc 
- dilatation morphologique
- Luminosité
- Gestion JPeG

Optionnel :
- Gestion différent format de fichier d'image (JPEG, PNG, BPM...)
- Color Mapping aléatoire 
- Filtre de Monte Carlo
- Filtrage aléatoire anisotrope

**Comment by GeraudSilvestri** [comment]
C'est tout bon 👍🏻

**Comment by ludelafo** [comment]
Bonjour @Nomeos, @Cat-du-Fromage,

Nous n'avons pas les accès à votre repo. Merci d'ajouter @hadrylouis, @GeraudSilvestri et moi-même pour que nous puissions y avoir accès pour le corriger.

**Reply by Nomeos** [reply]
Bonjour, je viens de vous ajouter ä l'instant, nous n'avions pas vu le message. Désolé pour le délai de réponse et de modification des droits.
